src/routes/transactions/entities/__tests__/transfer-transaction-info.builder.ts

```diff
@@ -18,6 +20,6 @@ export function transferTransactionInfoBuilder(): IBuilder<TransferTransactionIn
     )
     .with('transferInfo', {
       ...erc20TransferBuilder().build(),
-      type: 'ERC20_TRANSFER',
```

Comment:
This was incorrect. `ERC20_TRANSFER` is returned by the Transaction Service, but the `transferInfo` should return `ERC20`.

Reply:
I agree. Luckily the enum conversaion acted as a defacto test and caused a type error.